pkg/openfeature/util_test.go

```diff
@@ -0,0 +1,21 @@
+package openfeature
```

Comment:
maybe worth to move into a testing package?

Reply:
At least not for now. but this can be considered if we see this grow. Right now, this is small enough to live in this level